domain/robstoll-lib/atrium-domain-robstoll-lib-jvm/src/test/kotlin/ch/tutteli/atrium/assertions/filesystem/SymbolicLinkResolvingSpec.kt

```diff
@@ -29,8 +29,7 @@ import java.nio.file.Path
 import java.nio.file.Paths
 
 object SymbolicLinkResolvingSpec : Spek({
-    val tempFolder = TempFolder.perTest()
-    registerListener(tempFolder)
```

Comment:
@robstoll Can you help me with registerListener(). It was not accepting tempFolder as parameter.

Reply:
You don't require it anymore, `val tempFolder by memoizedTempFolder()` is good enough it registers itself

Reply:
Ok got it. I was assuming that is the reason for failing test cases.

Reply:
I am investigating why they fail right now. Locally it builds fine so maybe the problem has to do with skipping tests. Stay tuned